Cargo.lock

```diff
@@ -578,6 +579,12 @@ dependencies = [
  "winapi 0.3.8",
 ]
 
+[[package]]
+name = "dissimilar"
+version = "1.0.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "39de161cd2ebbd6e5783db53a82a47b6a47dcfef754130839603561745528b94"
+
```

Comment:
I'm glad that dissimilar doesn't have any dependencies.

Reply:
All credits go to @mookid at he suggested this awesome lib